resolvers/npm_cache/Cargo.toml

```diff
@@ -24,7 +24,7 @@ deno_path_util.workspace = true
 deno_semver.workspace = true
 deno_unsync = { workspace = true, features = ["tokio"] }
 faster-hex.workspace = true
-flate2 = { workspace = true, features = ["zlib-ng-compat"] }
+flate2 = { workspace = true, features = ["default"] }
```

Comment:
This change is one of the changes that I'll get rid of when rebasing and fixing the PR. Currently that's just to not require the CMake dependency when developing locally.